src/Axiom/Set/Map/Dec.agda

```diff
@@ -60,3 +61,37 @@ module Lookupᵐᵈ (sp-∈ : spec-∈ A) where
 
     aggregate₊ : FinSet (A × V) → Map A V
     aggregate₊ (_ , l , _) = foldl (λ m x → m ∪⁺ ❴ x ❵ᵐ) ∅ᵐ l
+
+    module _ {m m' : Map A V} where
+
+      dom-∪⁺⊆∪ : dom ((m ∪⁺ m') ˢ) ⊆ dom (m ˢ) ∪ dom (m' ˢ)
+      dom-∪⁺⊆∪ p = subst (_∈ dom (m ˢ) ∪ dom (m' ˢ))
+                         (Prelude.sym $ proj₁ (×-≡,≡←≡ $ proj₁ (proj₂ $ ∈-dom-∪⁺ p)))
+                         (proj₂ $ proj₁ (∈-dom-∪⁺ p))
+        where
+        f : Σ A (λ a → a ∈ dom (m ˢ) ∪ dom (m' ˢ)) → A × V
+        f (a , a∈) = a , (fold id id _◇_)(unionThese m m' a a∈)
```

Comment:
This `f` appears twice. How about you lift it out of this `where` block into a `private` block of the parent module and give it a slightly more descriptive name?